accounts/abi/bind/backend.go

```diff
@@ -74,6 +74,9 @@ type ContractTransactor interface {
 	// SuggestGasPrice retrieves the currently suggested gas price to allow a timely
 	// execution of a transaction.
 	SuggestGasPrice(ctx context.Context) (*big.Int, error)
+	// SuggestGasPriceInCurrency retrieves the currently suggested gas price to allow
+	// a timely execution of a transaction for an specific currency
+	SuggestGasPriceInCurrency(ctx context.Context, account common.Address) (*big.Int, error)
```

Comment:
```suggestion
	SuggestGasPriceInCurrency(ctx context.Context, currency *common.Address) (*big.Int, error)
```
Currency should be of type `*common.Address`. `nil` is the value in transactions that are paid in cGLD, and the zero address is an invalid currency value. Is it not possible to use that type in RPC?